### PR TITLE
Enhance flash message UI on incorrect credentials and logout #24

### LINKED ISSUE #24: Enhance flash message UI on incorrect credentials and logout
* Currently JS alert is used on events of incorrect credentials on login and logout events.

- [x] Create a favorable UI for the same, preferably a flash modal window.

### PR DESCRIPTION
**Pull request by anand-nakat**
Fixes #24 
1. Created Flash Messages for Validation alerts using $_SESSION[ ] array in PHP, and display them using Bootstrap Toasts.
2. Added Shadow to Logo Images on Index Page
image

### CODE REVIEW

File: login.php [hunk 1]
```diff
@@ -109,8 +140,52 @@
...
+	//Hide Toast & Toast Container if clicked outside of Toast
+	$(document).mouseup(event => {
+		if (!$toast.is(event.target) && $toast.has(event.target).length === 0) 
+		{
+			$('.toast').toast('hide');
+			$('.toast-container').fadeOut();
+			$("#uname").focus();
+		}
+	});
+
```
Comment (anand-nakat) [inline login.php:178]: Hide Toast if clicked outside of Toast

File: logout.php [hunk 2]
```diff
@@ -3,12 +3,12 @@
 session_start();
 
 if(isset($_SESSION['uname'])) {
-	session_destroy();
-	echo "<script> alert('" . $_SESSION['uname'] . " logged out succesfully!') </script>";
+	$_SESSION['loggedOut']=true;
+	$_SESSION['message']=$_SESSION['uname']." Logged Out succesfully!";
 	echo "<script> location.href='login.php' </script>";
 }
```
Comment (anand-nakat) [inline logout.php:9]: Session is destroyed in login.php

File: login.php [hunk 3]
```diff
@@ -109,8 +140,52 @@
...
+			{	
+				echo("$('.toast-container').fadeIn();$('.toast').toast('show');");  
+				unset($_SESSION['message']);
+				
+				//Destroy Session after Displaying 'Log out' Message
+				if(isset($_SESSION['loggedOut']))
+				{
+					if($_SESSION['loggedOut'] === true)
+						session_destroy();
+				}
```
Comment (anand-nakat) [inline login.php:162]: Destroy Session if logout.php is executed

File: login.php [hunk 4]
```diff
@@ -54,42 +62,65 @@
...
+			<div class="toast-header">
+				<strong class="mr-auto text-black">CyanoKhoj.</strong>
+				<button type="button" id="toast-close" class="ml-2 mb-1 close" data-dismiss="toast" style="font-size:3rem;">&times;</button>
+			</div>
+			<div class="toast-body text-black text-capitalize" style="font-weight: 600;background: #DD9106;">
+				<?php if(isset($_SESSION['message']))
+					echo($_SESSION['message']); ?>
+			</div>
+		</div>
+	</div>
```
Comment (anand-nakat) [inline login.php:118]: Toast Container

File: index.php [hunk 5]
```diff
@@ -3,6 +3,7 @@
 
 if (isset($_SESSION['uname']) == FALSE) {
   echo "<script> location.href='login.php' </script>";
+  $_SESSION['message']="Please Log in!";
```
Comment (Chintan2108) [inline index.php]: @anand-nakat Can you provide a corresponding event snapshot for this message?
Is the 'message' key in the SESSION array needed?

File: index.php [hunk 6]
```diff
@@ -3,7 +3,6 @@
 
 if (isset($_SESSION['uname']) == FALSE) {
   echo "<script> location.href='login.php' </script>";
-  $_SESSION['message']="Please Log in!";
```
Comment (anand-nakat) [inline index.php]: Removed Message from index.php

**Review by Chintan2108 — APPROVED** [review]
@anand-nakat Changes approved for mergning

**Comment by Chintan2108** [comment]
give @anand-nakat 25 points

**Pull request merged**